Makefile

```diff
@@ -46,8 +46,12 @@ tar: youtube-dl.tar.gz
 pypi-files: youtube-dl.bash-completion README.txt youtube-dl.1 youtube-dl.fish
 
 youtube-dl: youtube_dl/*.py youtube_dl/*/*.py
-	zip --quiet youtube-dl youtube_dl/*.py youtube_dl/*/*.py
-	zip --quiet --junk-paths youtube-dl youtube_dl/__main__.py
+	mkdir -p zip
+	cp -a --parent youtube_dl/*.py youtube_dl/*/*.py zip/
+	touch -d@946684800 zip/youtube_dl/*.py zip/youtube_dl/*/*.py
```

Comment:
The timestamp shouldn't be hard-coded. ```SOURCE_DATE_EPOCH``` environment variable seems a good choice. https://reproducible-builds.org/docs/timestamps/